eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/tellraw/TellRawCommand.java

```diff
@@ -0,0 +1,41 @@
+package com.eternalcode.core.feature.essentials.tellraw;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.notice.NoticeService;
+import com.eternalcode.core.notice.NoticeTextType;
+import com.eternalcode.core.viewer.Viewer;
+import dev.rollczi.litecommands.argument.Arg;
+import dev.rollczi.litecommands.argument.joiner.Joiner;
+import dev.rollczi.litecommands.command.execute.Execute;
+import dev.rollczi.litecommands.command.permission.Permission;
+import dev.rollczi.litecommands.command.route.Route;
+import org.bukkit.entity.Player;
+
+@Route(name = "tellraw")
+@Permission("eternalcore.tellraw")
+public class TellRawCommand {
+
+    private final NoticeService noticeService;
+
+    public TellRawCommand(NoticeService noticeService) {
+        this.noticeService = noticeService;
+    }
+
+    @Execute
+    @DescriptionDocs(description = "Send player a message without any prefixes etc.", arguments = "<player> <notice_type> <message>")
```

Comment:
w tym `@DecriptionDocs` to jest nie poprawnie napisane. Według mnie powinno być `Send a message to the player without any prefixes etc.`